content/zh-cn/docs/_index.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
None of the links in the Chinese index page link to the Chinese pages. Every time a user clicks on a link, it will lead them to the English pages.

Also, pages that are not translated should still have the English title (and link) in the index page.

Reply:
Hi,

I'm working on translating those pages now...